cloud/build_gnu_images.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
Maybe more this, and the other .sh scripts in `cloud/`, into `cloud/scripts/` to keep things a bit more clean?